root/src/main/java/com/example/servingwebcontent/ConnectDatabase.java

```diff
@@ -15,31 +15,66 @@ public ConnectDatabase (String database, String user, String pass) {
 		}
 	}
 
-	public ResultSet queryDB (String query) {
+	public ResultSet getMovie (String id) {
 		ResultSet rs = null;
+		String query = "select * from movies where id=?";
 		try {
-			Statement stmt = con.createStatement();
-			rs = stmt.executeQuery(query);
+			PreparedStatement stmt = con.prepareStatement(query);
+			stmt.setString(1, id);
+			rs = stmt.executeQuery();
 		} catch (SQLException e) { 
-			throw new RuntimeException("failed to query db" ,e); 
+			throw new RuntimeException("failed to query db", e); 
+		}
+
+		return rs;
+	}
+
+	public ResultSet updateMovie (String id, String column, String value) {
+		ResultSet rs = null;
+		String query = "";
+		if (column.equals("type")) {
+			query = "update movies set type = ? where id = ?";
+		}
+		else if (column.equals("name")) {
```

Comment:
everyone in the industry puts "else" on the same line as the curly brace which saves a line. Try formatting it like that to get used to it.

Reply:
These if/else blocks will no longer exist but this is for future reference as my above comment about using the movie object as input will change this